src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -573,31 +596,34 @@ def create_lv(name, group, extents=None, size=None, tags=None, uuid_name=False):
         'lockbox': 'ceph.lockbox_device',  # XXX might not ever need this lockbox sorcery
     }
     if size:
-        process.run([
+        command = [
             'lvcreate',
             '--yes',
             '-L',
             '%s' % size,
             '-n', name, group
-        ])
+        ]
     elif extents:
-        process.run([
+        command = [
             'lvcreate',
             '--yes',
             '-l',
             '%s' % extents,
             '-n', name, group
-        ])
+        ]
     # create the lv with all the space available, this is needed because the
     # system call is different for LVM
     else:
-        process.run([
+        command = [
             'lvcreate',
             '--yes',
             '-l',
             '100%FREE',
             '-n', name, group
-        ])
+        ]
+    if pv is not '':
```

Comment:
The tox tests are failing because of this line. The linter complains about `F632 use ==/!= to compare str, bytes, and int literals` This considered bad form since it relies on the cython implementation to do the right thing. Other interpreters might might treat a `str` comparison with `is` differently.